map-with-map-settings/demo.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * Boilerplate map initialization code starts below:
+ */
+function getGeoJSONData () {
+  // Create GeoJSON reader which will download the specified file.
+  // Shape of the file was obtained by using HERE Geocoding and Search API.
+  // It is possible to customize look and feel of the objects.
+  var reader = new H.data.geojson.Reader('data/berlin.json', {
+    // This function is called each time parser detects a new map object
+    style: function (mapObject) {
+      // Parsed geo objects could be styled using setStyle method
+      if (mapObject instanceof H.map.Polygon) {
+        mapObject.setStyle({
+          fillColor: 'rgba(255, 0, 0, 0.5)',
+          strokeColor: 'rgba(0, 0, 255, 0.2)',
+          lineWidth: 3
+        });
+      }
+    }
+  });
+
+  // Start parsing the file
+  reader.parse();
+
+  // return layer which shows GeoJSON data on the map
+  return reader.getLayer();
+}
+
+// Step 1: initialize communication with the platform
+// In your own code, replace variable window.apikey with your own apikey
+
+var platform = new H.service.Platform({
+  apikey: window.apikey
+});
+
+// Step 2: specify engine type. In this example, we use HARP rendering engine, which used to Render Hybrid Layers.
+// HARP engine is not the default engine, and it's not included in the mapsjs-core.js module.
+// To use this engine, you need to include mapsjs-harp.js file to your HTML page
+var engineType = H.Map.EngineType['HARP'];
+
+// Step 3: create the style object from the style configuration
+// exported from the HERE Style Editor. The argument is a style path
+var style = new H.map.render.harp.Style('https://heremaps.github.io/maps-api-for-javascript-examples/change-harp-style-at-load/data/night.json');
+
+// Step 4: create a layer with the style object:
+var vectorLayer = platform.getOMVService().createLayer(style, { engineType });
```

Comment:
We are creating already all the default layers in line 65. What's the need to create a layer here?